pkg/process/config/yaml_config.go

```diff
@@ -254,6 +255,16 @@ func (a *AgentConfig) LoadProcessYamlConfig(path string) error {
 		}
 	}
 
+	if k := key(ns, "queue_bytes"); config.Datadog.IsSet(k) {
+		if queueBytes := config.Datadog.GetInt(k); queueBytes > math.MaxInt32 {
+			log.Warnf("Invalid queue_bytes setting, value exceeds maximum: %d", math.MaxInt32)
+		} else if queueBytes <= 0 {
+			log.Warn("Invalid queue_bytes setting, value should be > 0")
```

Comment:
Can we specify what we're defaulting to in these two cases?

Also thoughts on just merging these cases with a generic `"invalid queue_bytes setting: %d, defaulting to: %d"` message?